javascripts/terms-and-conditions.js

```diff
@@ -57,7 +57,7 @@ app.termsAndConditions = {
 
       // Inform GTM that we have requested a product download
       window.dataLayer = window.dataLayer || [];
-      window.dataLayer.push({ 'product_download_file_name' : $.encoder.encodeForJavascript(tcDownloadFileName).replace(/\x2D/g, '-') });
```

Comment:
It looks like the problem in the end was encoding the tcDownloadFileName again using $.encoder.encodeForJavascript